Makefile

```diff
@@ -37,11 +38,17 @@ IMAGE_SUFFIX ?= $(strip )
 
 # REGISTRY ?= docker.io/library
 # Container registry for target images.
-REGISTRY ?= docker.io/library
+REGISTRY ?= docker.io/caicloud
 
-# Container registry for base images.
+# Container registry for build base images.
 BASE_REGISTRY ?= docker.io/library
 
+# Container registry for cyclone target base images.
+CYCLONE_BASE_REGISTRY ?= docker.io/caicloud
```

Comment:
I think not need this, can reuse the `REGISTRY`, as the target images and base images will been in the same registry.